helm/prometheus-rules/templates/alerting-rules/prometheus-agent.rules.yml

```diff
@@ -74,5 +73,4 @@ spec:
         cancel_if_cluster_is_not_running_prometheus_agent: "true"
         cancel_if_cluster_status_creating: "true"
         cancel_if_cluster_status_deleting: "true"
-        cancel_if_outside_working_hours: "true"
```

Comment:
This one is quite young and can be very noisy, so I'm not sure we should unleash it so soon.
It's been fixed on last Friday, and we immediately had to silence it on clusters that use heavy autoscaling.

Reply:
Sure so https://github.com/giantswarm/prometheus-rules/pull/911/files should have fixed the paging issues that happened during the night. We need to rework it a bit but we cannot not be paged when a Prometheus Agent is down though

Reply:
Sure.
So I think for now it would be best to have:
- `PrometheusAgentFailing` pages at night
- `PrometheusAgentShardsMissing` keeps it quiet at night (for now)

Reply:
Latest commit should address that until we fixed the shard computation